src/scripts/main.js

```diff
@@ -1,3 +1,25 @@
 'use strict';
 
-// write your code here
+const textCollection = document.querySelectorAll('.population');
+
+const arr = [...textCollection];
+const textArray = [];
+
+for (const item of arr) {
+  const textData = item.textContent;
+  const digitData = Number(textData.split(',').join(''));
+
+  textArray.push(digitData);
+}
```

Comment:
I case that you are not going to use array methods. There is no sense to make an Array from collection  
```suggestion
const textArray = [];

for (const item of textCollection) {
  const textData = item.textContent;
  const digitData = Number(textData.split(',').join(''));

  textArray.push(digitData);
}
```